upgrading/extending-daml-models/0_intro.md

```diff
@@ -0,0 +1,18 @@
+Database schemas tend to evolve over time. Your business might change, the data that you collect
+might need to be extended or your access rights might need updating. For example, you might decide
+to interact with a new partner and share your database, but the access rights need to be different
+from what you have deployed for your other partners. Or maybe you decide to share parts of your data
+of a running ledger with the public for reading but not writing.
+
+In this first article on DAML ledger upgrades you learn how to extend your DAML model and ship this
+extension as a DAML package to a running ledger. This package will depend on the already deployed
+DAML packages on the ledger. As a running example, we will extend the social network you build in
+the [Getting Started Guide](https://daml.com/learn/getting-started/) with a forum.
+
+In the follow up article on DAML upgrades, you will learn how to make changes to an already deployed
+DAML package and how to migrate the data to your new model.
+
+You can find more documentation on upgrading DAML models and ledgers
+[here](https://docs.daml.com/upgrade/index.html).
+
+![AsciiCinema](assets/upgrading-daml-models.gif)
```

Comment:
This won't work in embedded mode.

Reply:
The problem is that the iframe that embeds the scenario in daml.com/learn points to

https://katacoda.com/embed/daml/...

But your assets live at

https://katacoda.com/daml/...

Reply:
nvm, Katacoda seem to have fixed that issue by also making assets available under the /embed/ path now. That didn't work when I created the original GSG.